### PR TITLE
adding IAM role to use with control1-machine and copy formplayer logs

### PR DESCRIPTION
**Pull request by prem-fissionhq**
https://dimagi-dev.atlassian.net/browse/SAAS-12190
##### ENVIRONMENTS AFFECTED

None.

We would like to attach this role to Control1-machine in production and run the python script which copies current formplayer logs into hive format.

@dannyroberts could you please create this IAM role in production. thanks

### CODE REVIEW

**Comment by prem-fissionhq** [comment]
Included iam role and policy under IAM module instead of under server module. Fixing it.

**Review by dannyroberts — APPROVED** [review]

File: src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2 [hunk 1]
```diff
@@ -207,6 +207,7 @@ module "openvpn" {
 module "Users" {
   source = "./modules/iam"
```
Comment (dannyroberts) [inline src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2:208]: @prem-fissionhq pointed out online that server.iam would be a better place for these changes, so I'll review again after that

File: src/commcare_cloud/terraform/modules/iam/main.tf [hunk 2]
```diff
@@ -24,3 +24,4 @@ resource "aws_iam_policy" "force_mfa" {
   description = "This policy allows users to manage their own passwords and MFA devices but nothing else unless they authenticate with MFA"
   policy = "${file("${path.module}/EnforceMFAPolicy.json")}"
 }
+
```
Comment (dannyroberts) [inline src/commcare_cloud/terraform/modules/iam/main.tf:27]: nit: this looks like a stray blank line at the end of the file / unrelated change

**Review by dannyroberts — APPROVED** [review]
Looks good
